### DISCUSSION TITLE
"LOVE SFL" statue on Helios Island!

### DISCUSSION THREAD

**Discussion by digital-farms** [sunflower-land/sunflower-land#1983] [Art] [answered]
## What do you think of the idea of installing such a statue on Helios Island instead of the usual one taken from the Daniel Diggle Asset Pack?

**_*Added below is a modified Helios Island Asset and two versions of the statue itself_**
![LOVE-SFL-STATUE-ON-HELIOS](https://user-images.githubusercontent.com/105886943/210786457-c74883e1-67ea-443b-85e4-19480c774a25.png)

**The modified Helios Island Asset, without resizing:**
![Helios](https://user-images.githubusercontent.com/105886943/210788212-cc7195f2-13e4-4066-bac7-fa61a8a69c8d.png)

**Asset "LOVE SFL" statues:**
![LOVE_SFL_NFT](https://user-images.githubusercontent.com/105886943/210788552-3f9185dc-730e-4d72-a28d-3c217511889e.gif) - GIF
![LOVE_SFL_NFT_static](https://user-images.githubusercontent.com/105886943/210788558-ddd4d231-86db-43bd-9512-95d502ac072c.png) - Static

`land id: 21637`
`discord: KOTOB#4391`

**Comment by IvanSkazkin** [comment]
Love it is

**Comment by sabryder** [comment]
i love it🤩🤩🤩✌🏻

**Comment by adamhannigan** [comment, marked as answer]
Nice work @digital-farms 👏 

Aesthetic wise, I feel the current statue is more aligned with the style of Helios.

However, I do believe that an SFL statue is more relevant for the game + SFL players. This may be a better candidate for one of our upcoming islands or community islands

**Reply by digital-farms** [reply]
Maybe you'll like this😊

![world](https://user-images.githubusercontent.com/105886943/211938827-068daa77-bd8b-42a2-8ecc-6c45f81369b1.png)

**Reply by digital-farms** [reply]
Or like this😁
Sorry if I'm too pushy, I won't do it again)

![world_1](https://user-images.githubusercontent.com/105886943/212183770-cf000198-fd66-413f-a2e9-1d89e9e931cc.png)
